### PR TITLE
Compose v1.0.2

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt [hunk 1]
```diff
@@ -123,7 +123,7 @@ private fun ButtonPreview() {
 private fun RectangleButtonPreview() {
     JetsnackTheme {
         JetsnackButton(onClick = {}, shape = RectangleShape) {
-            Text(text = "Demo")
-        }
+        Text(text = "Demo")
+    }
```
Comment (manuelvicnt) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt:127]: Revert back?

Comment (JolandaVerhoef) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt:127]: This is actually done by SpotlessApply. Without it the build fails. I'll find or raise a bug for ktlint or spotless.

Comment (JolandaVerhoef) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt:127]: Found it! https://github.com/pinterest/ktlint/issues/1221

Comment (manuelvicnt) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt:127]: Cool! Should we revert the version update back until it's fixed?

Comment (JolandaVerhoef) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt]: Done

**Review by manuelvicnt — APPROVED** [review]

**Pull request merged**